SoundSwitch/UI/Forms/AddProfile.Designer.cs

```diff
@@ -0,0 +1,259 @@
+            this.addProfileGroupBox = new System.Windows.Forms.GroupBox();
+            this.recordingRemoveButton = new System.Windows.Forms.Button();
+            this.playbackRemoveButton = new System.Windows.Forms.Button();
+            this.hotKeyTextBox = new SoundSwitch.UI.UserControls.HotKeyTextBox();
+            this.recordingLabel = new System.Windows.Forms.Label();
+            this.playbackLabel = new System.Windows.Forms.Label();
+            this.recordingComboBox = new SoundSwitch.UI.UserControls.IconTextComboBox();
+            this.playbackComboBox = new SoundSwitch.UI.UserControls.IconTextComboBox();
+            this.selectProgramButton = new System.Windows.Forms.Button();
+            this.programLabel = new System.Windows.Forms.Label();
+            this.programTextBox = new System.Windows.Forms.TextBox();
+            this.hotKeyLabel = new System.Windows.Forms.Label();
+            this.nameLabel = new System.Windows.Forms.Label();
+            this.nameTextBox = new System.Windows.Forms.TextBox();
+            this.selectProgramDialog = new System.Windows.Forms.OpenFileDialog();
+            this.createButton = new System.Windows.Forms.Button();
+            this.addProfileGroupBox.SuspendLayout();
+            this.SuspendLayout();
+            // 
+            // addProfileGroupBox
+            // 
+            this.addProfileGroupBox.Anchor = ((System.Windows.Forms.AnchorStyles) (((System.Windows.Forms.AnchorStyles.Top | System.Windows.Forms.AnchorStyles.Left) | System.Windows.Forms.AnchorStyles.Right)));
+            this.addProfileGroupBox.AutoSizeMode = System.Windows.Forms.AutoSizeMode.GrowAndShrink;
+            this.addProfileGroupBox.Controls.Add(this.recordingRemoveButton);
+            this.addProfileGroupBox.Controls.Add(this.playbackRemoveButton);
+            this.addProfileGroupBox.Controls.Add(this.hotKeyTextBox);
+            this.addProfileGroupBox.Controls.Add(this.recordingLabel);
+            this.addProfileGroupBox.Controls.Add(this.playbackLabel);
+            this.addProfileGroupBox.Controls.Add(this.recordingComboBox);
+            this.addProfileGroupBox.Controls.Add(this.playbackComboBox);
+            this.addProfileGroupBox.Controls.Add(this.selectProgramButton);
+            this.addProfileGroupBox.Controls.Add(this.programLabel);
+            this.addProfileGroupBox.Controls.Add(this.programTextBox);
+            this.addProfileGroupBox.Controls.Add(this.hotKeyLabel);
+            this.addProfileGroupBox.Controls.Add(this.nameLabel);
+            this.addProfileGroupBox.Controls.Add(this.nameTextBox);
+            this.addProfileGroupBox.Location = new System.Drawing.Point(14, 14);
+            this.addProfileGroupBox.Name = "addProfileGroupBox";
+            this.addProfileGroupBox.Size = new System.Drawing.Size(440, 282);
+            this.addProfileGroupBox.TabIndex = 0;
+            this.addProfileGroupBox.TabStop = false;
+            this.addProfileGroupBox.Text = "Profiles";
```

Comment:
Not localized and the group box is not for profiles it's for a profile.

Also check that any other string that may be not localized.